content/v1.16/cli/command-reference.md

```diff
@@ -828,7 +828,8 @@ Configuration/platform-ref-aws                             v0.9.0    True
 
 The `crossplane beta validate` command validates 
 [compositions]({{<ref "../concepts/compositions">}}) against provider or XRD 
-schemas using the Kubernetes API server's validation library.
+schemas using the Kubernetes API server's validation library 
```

Comment:
Something I realized since we synced on Monday is that we should only make this docs change if the code change itself is going to be back ported to previous releases in a patch release (e.g. a v1.16.1).

We'll wait to ship this functionality in v1.17, so there's no need to make any docs changes for anywhere but in the `master` directory. You can remove this change 😇